rasa_core/registry.py

```diff
@@ -0,0 +1,38 @@
+"""This module imports all of the components. To avoid cycles, no component 
+should import this in module scope."""
+
+import logging
+import typing
+from typing import Any, Dict, List, Optional, Text, Type
+import importlib
+
+from rasa_core import utils
+from rasa_core.policies.keras_policy import KerasPolicy
```

Comment:
let's import all the policies/featurizers within the function of policy_from_module_path